### PR TITLE
Implement Utils classes in Kotlin.

### PR DESCRIPTION
**Pull request by Isira-Seneviratne**
Issue: https://issues.apache.org/jira/projects/LANG/issues/LANG-1538

### CODE REVIEW

**Comment by Isira-Seneviratne** [comment]
Maybe it could be implemented as a separate library?

File: pom.xml [hunk 1]
```diff
@@ -74,7 +71,7 @@
       <name>Henri Yandell</name>
       <id>bayard</id>
       <email>bayard@apache.org</email>
-      <organization />
+      <organization/>
```
Comment (garydgregory) [inline pom.xml:74]: @Isira-Seneviratne 
Please do not change this formatting.

File: pom.xml [hunk 2]
```diff
@@ -553,6 +550,12 @@
       <scope>test</scope>
     </dependency>
 
+    <!-- Kotlin -->
+    <dependency>
+      <groupId>org.jetbrains.kotlin</groupId>
+      <artifactId>kotlin-stdlib-jdk8</artifactId>
+      <version>${kotlin.version}</version>
+    </dependency>
   </dependencies>
```
Comment (garydgregory) [inline pom.xml:559]: -1, that's a bridge too far IMO. If we want to create a Kotlin Commons Lang, we can talk about that on the developer's mailing list...

File: pom.xml (hunk 1 above)
Comment (Isira-Seneviratne) [reply pom.xml:74]: That was formatted by IntelliJ.

Comment (garydgregory) [reply pom.xml:74]: It does not matter who formatted what, it's still your PR ;-)

**Pull request closed without merging**